scenes/PlayerSelection.tscn

```diff
@@ -0,0 +1,710 @@
+[gd_scene load_steps=12 format=2]
+
+[ext_resource path="res://sprites/Background_gradient.png" type="Texture" id=1]
+[ext_resource path="res://sprites/Title_logo.png" type="Texture" id=2]
+[ext_resource path="res://sprites/Player_Orange.png" type="Texture" id=3]
+[ext_resource path="res://sprites/Player_Green.png" type="Texture" id=4]
+[ext_resource path="res://sprites/Player_Purple.png" type="Texture" id=5]
+[ext_resource path="res://sprites/Player_Yellow.png" type="Texture" id=6]
+[ext_resource path="res://sprites/Player_Red.png" type="Texture" id=7]
+[ext_resource path="res://sprites/Player_Blue.png" type="Texture" id=8]
+[ext_resource path="res://sprites/AI_Mark.png" type="Texture" id=9]
+[ext_resource path="res://sprites/Human_Mark.png" type="Texture" id=10]
+
+[sub_resource type="GDScript" id=1]
+script/source = "extends Node2D
+
+var PLAYER_SETTINGS = [
+	[ENUMS.PLAYER_TYPE.HUMAN, ENUMS.AI_STRATEGY.SOLO, false, false],
+	[ENUMS.PLAYER_TYPE.HUMAN, ENUMS.AI_STRATEGY.SOLO, false, false],
+	[ENUMS.PLAYER_TYPE.HUMAN, ENUMS.AI_STRATEGY.SOLO, false, false],
+	[ENUMS.PLAYER_TYPE.HUMAN, ENUMS.AI_STRATEGY.SOLO, false, false]]
+
+# ================================================== #
+# ================================================== #
+
+var standardButton = Color.gray
+var highlightedButton = Color.greenyellow
+
+var optionalButtonRed = Color.webgray
+var optionalButtonBlue = Color.webgray
+var optionalButtonGreen = Color.webgray
+var optionalButtonYellow = Color.webgray
+
+func object_features_init():
```

Comment:
Make sure that when exiting the game with `ESC` and returning to this scene you still have correct player settings selected - IMO it would be better to remember the settings the game was started with and because I suggest using just one instance of this player settings array it's really easy to do